packages/ui/src/components/alert/Alert.test.tsx

```diff
@@ -2,18 +2,26 @@ import { render, screen } from '@testing-library/react'
 
 import { Alert } from './'
 
+beforeAll(() => {
+  global.ResizeObserver = class {
+    observe() {}
+    unobserve() {}
+    disconnect() {}
+  }
+})
```

Comment:
This is likely to be an issue in other places. Can we move this block to the test setup file?